tests/unit/services/utils.spec.js

```diff
@@ -0,0 +1,27 @@
+import {
+  generateUniqueNumbers,
+} from '@/services/utils';
+
+
+describe('Utils service', () => {
+  describe('generateRandomNumbers', () => {
+    it('returns array of random unique integers on open interval', () => {
+      const max = 5;
+      const min = 1;
+      const amount = 3;
+      const randomInts = generateUniqueNumbers(amount, min, max);
+      expect(randomInts.length).toBe(amount);
+      expect(Math.max(...randomInts)).toBeLessThanOrEqual(max);
+      expect(Math.min(...randomInts)).toBeGreaterThanOrEqual(min);
+      expect(new Set(randomInts).size).toBe(randomInts.length);
```

Comment:
This is a check to validate numbers are indeed unique. Is it clear or a comment would be welcomed?

Reply:
:+1: 

Which `expects` do you mean? One is for `max`, one for `min` and one for `unique. `